### DISCUSSION TITLE
[@types/lodash]

### DISCUSSION THREAD

**Discussion by zepplock** [DefinitelyTyped/DefinitelyTyped#65088] [Issues with a @types package]
Hello, I have a weird case with lodash types after going from `@types/lodash: ^4.14.180`, to `@types/lodash: ^4.14.192`. Looks like `groupBy` 1st parameter changed from `string` to `_.ValueIteratee<MyType>` where MyType is a type of the collection `groupBy` is applied to. As a result code with nested keys is no longer type safe, for example: for this object/collection ```{"a": {"b": "c"}}```, `.groupBy('a.b')` is no longer applicable since only 'a' is a valid type.

[a comment by an automated account is not shown]

**Comment by zepplock** [comment]
I think there's an ongoing effort to revert it back: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/64924

**Reply by jakebailey** [reply]
This has been merged.
